Shared/View/VoteView.swift

```diff
@@ -8,28 +8,50 @@
 import SwiftUI
 
 struct VoteView: View {
-    // Hardcoded values to test
-    @State private var tempActivty = Activity(
-        id: "123",
-        title: "What movie should we watch tonight?",
-        category: Category.movie,
-        // FIXME: this is not a valid user ID
-        // TODO: request renaming by backend if this is confusing
-        author: "mightyDonut"
-    )
+    @EnvironmentObject var user: UserViewModel
+    @EnvironmentObject var activity: ActivityViewModel
+    @EnvironmentObject var option: OptionViewModel
```

Comment:
These two are not really suitable to pass down as environment object. They are more so designed to observe a specific activity. So instead you'll have something like 

```swift
@ObservedObject var activityViewModel = ActivityViewModel(id: "AhdjSLOlTJqb68aXBAWn") 
```

then you can get an always up to date activity with

```swift
activityViewModel.activity
```